test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -22,6 +22,7 @@
 from helpers.StacktraceHelper import getCoredumps, generateStacktrace
 from helpers.SyncHelper import closeSocketConnection, clearWaitedAfterSync
 from helpers.SpaceHelper import delete_project_spaces
+from helpers.api.Provisioning import delete_groups
```

Comment:
CI complains:
```
ImportError: cannot import name 'delete_groups' from 'helpers.api.Provisioning' (/drone/src/test/gui/shared/scripts/helpers/api/Provisioning.py)
```
should be
```suggestion
from helpers.api.Provisioning import delete_created_groups
```